src/test/java/lotto/LottoEarningCalculatorTest.java

```diff
@@ -7,88 +7,101 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class LottoEarningCalculatorTest {
 
-    LottoTicket winningTicket = new LottoTicket(List.of(1, 2, 3, 4, 5, 6));
+    final LottoTicket winningTicket = new LottoTicket(Stream.of(1, 2, 3, 4, 5, 6).map(LottoNumber::new).collect(Collectors.toList()));
+    final LottoNumber bonus = new LottoNumber(7);
```

Comment:
```suggestion
    private final LottoTicket winningTicket = new LottoTicket(Stream.of(1, 2, 3, 4, 5, 6).map(LottoNumber::new).collect(Collectors.toList()));
    private final LottoNumber bonus = new LottoNumber(7);
```

중요하지는 않을 수도 있지만 테스트 코드에서 역시 접근 제어자에 신경 써 주세요 😅